beacon-chain/blockchain/service.go

```diff
@@ -93,30 +89,13 @@ func (c *ChainService) Start() {
 func (c *ChainService) Stop() error {
 	defer c.cancel()
 	log.Info("Stopping service")
-	log.Info("Persisting current active and crystallized states before closing")
```

Comment:
I removed this because the state's always written to disk. I don't think it's a good idea to depend on the `Stop` function to persist state to disk. If the application crashes, state won't be saved to disk and the database is left in an inconsistent state.